include/u-boot.mk

```diff
@@ -88,7 +111,9 @@ define Build/Configure/U-Boot
 		+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) $(UBOOT_CONFIGURE_VARS) oldconfig)
 endef
 
-DTC=$(wildcard $(LINUX_DIR)/scripts/dtc/dtc)
+ifndef UBOOT_USE_INTREE_DTC
```

Comment:
The `DTC`enviroemnt variable is already used by some other u-boot builds like `uboot-sifiveu`, `uboot-sunxi` and maybe `uboot-fritz4040`, `uboot-mediatek`, please set `UBOOT_USE_INTREE_DTC` in these builds already in this commit.

Reply:
Okay I updated `uboot-mediatek`, `uboot-sifiveu` and `uboot-sunxi`.
For `uboot-fritz4040` I'm not sure, the Makefile seems like to simply export the `DTC` variable.